<?xml version="1.0" ?>
<launch>

  <arg name="mode" default="sim"/>
  <arg name="namespace" default="$(arg namespace)"/>

  <node type="bathy_slam_node" pkg="bathy_slam" name="bathy_slam_node" output="screen">
    
    <param name="mbes_pings"  value="/gt/mbes_pings" />
    <param name="odom_gt"  value="/gt/odom" />
    <param name="world_frame" value="world" />
    <param name="map_frame" value="map" />
    <param name="odom_frame" value="odom" />
    <param name="base_link" value="$(arg namespace)/base_link" />
    <param name="mbes_link" value="$(arg namespace)/mbes_link" />

  </node>
  
</launch>